gef-ui/server/controllers/amendments/what-needs-to-change/post-what-needs-to-change.ts

```diff
@@ -12,6 +12,7 @@ import { validationErrorHandler } from '../../../utils/helpers';
 export type PostWhatNeedsToChangeRequest = CustomExpressRequest<{
   params: { dealId: string; facilityId: string; amendmentId: string };
   reqBody: { amendmentOptions: string[] };
+  query: { change?: 'true' };
```

Comment:
```suggestion
  query: { change?: string };
```

I would suggest keeping it as a string please.